pkg/kubectl/cmd/get/get_test.go

```diff
@@ -1649,5 +1648,5 @@ func watchBody(codec runtime.Codec, events []watch.Event) io.ReadCloser {
 			panic(err)
 		}
 	}
-	return json.Framer.NewFrameReader(ioutil.NopCloser(buf))
+	return ioutil.NopCloser(buf)
```

Comment:
Hrm.  Now I'm scared.  It shouldn't matter for json, but if we had a protobuf serialization I think this would break.

I think that this is a bug in the frame reader

Reply:
I think this should have return empty

Reply:
this is returning the reader that populates the response body. the client stacks another framereader on top of this, which didn't play nicely with each other. we never noticed because we didn't return responses big enough to exceed the buffer size.

Reply:
ok